### ISSUE TITLE
Item is not moving after changing name

### ISSUE THREAD

**Issue by jacekblach** [files-community/Files#4214]
**Is your feature request related to a problem?**
A clear description of what the problem is.

**Describe the solution**
A clear description of what you want.

**Describe alternatives you've considered**
A clear description of any alternative solutions or features you've considered.

**Additional comment**
Add any other comment or screenshots about the feature request here. What device type/UI layout and app version?

**Comment by jacekblach** [comment]
In Q-dir is an option that item is not moving when I am changing name, or I am pasting not object in particular place. And it is arranging only when I refresh the page. For me it is the best option in file managers ever. I think it will be very useful if you do thing in 'Files". Your file manager is very promising.

**Comment by Spid3rrr** [comment]
I can start working on this @yair100 .

**Comment by yaira2** [comment]
@Spid3rrr any luck?

**Comment by noblemd00** [comment]
This should be an OPTION NOT A FEATURE - by user selection - ALWAYS reorder the list of files after rename - option "do not reorder after rename" or "reorder after rename". In a folder with a long list of folders/files reordering files after rename is a must. The list should focus on the file (to be) renamed and after it should be visible in its proper sorted position in the file list and displayed in the center (not top or bottom) of the current sorted view.

**Comment by Spid3rrr** [comment]
> @Spid3rrr any luck?

My apologies, I have been unable to work on it for quite a while. I think someone else might need to take it.

**Comment by EchoARC9** [comment]
I agree with @noblemd00. I prefer it to reorder automatically after a rename, so just switching it would be hurtful to anyone like me.

**Comment by rihards-simanovics** [comment]
Will just add my side problem which could be related to this one. In column tree layout, when a file such as excel spreadsheet changes in the parent dir and you are working with a file in a child dir, the focus changes to the excel file in the parent dir.

I noticed this bug when doing my bookkeeping for last year, whenever I previewed a pdf invoice with QuickLook in the child dir, and excel spreadhseet got changed due to me changing value in there, the focus would jump to excel spreadsheet closing the child dir and switching the preview to excel thumbnail.

**Comment by rihards-simanovics** [comment]
And this is how the issue comes across in case it helps:
https://github.com/files-community/Files/assets/52106095/77d40610-e0d5-4ad7-a1d7-3e02ed765e8d

**Comment by Josh65-2201** [comment]
That is a different issue #14011

**Comment by yaira2** [comment]
> That looks like a different issue #14011

They are similar, in both cases the file list is updating and causing the wrong item to be selected.

**Comment by yair100** [comment]
@jacekblach this issue will be resolved in the next release.
